ComfySoda_Unity2018.2.7f1/Assets/Scripts/ComySodaEngine/EnemyController.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Unity;
+using UnityEditor;
+using UnityEngine;
+
+
+namespace cs
+{
+    public class EnemyController : MonoBehaviour
+    {
+
+        #region Static referencing
+        public static EnemyController enemyController = null;
+        private void Awake()
+        {
+            Debug.Assert(enemyController == null, "Error: There should only be one enemy controller assigned to \"EnemyController.enemyController\" at a time");
+            enemyController = this;
+        }
+
+        private void OnDestroy()
+        {
+            Debug.Assert(enemyController == this, "Error: The enemy controller assigned to \"EnemyController.enemyController\" does not equal: " + gameObject);
+            enemyController = null;
+        }
+        #endregion
+
+
+        #region Enemy distance
+        [SerializeField]
+        public float CLOSE = 7;
+        [SerializeField]
+        public float MID = 12;
```

Comment:
public on any Serializable class (which Monobehaviours always are) is serializable by default. This is redundant.

Reply:
yeah, that was left over code from testing I missed when it was private.  I've removed it due to redundancy as per your suggestion.